src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubExtensionConfigProvider.cs

```diff
@@ -6,16 +6,17 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.EventHubs;
 using Microsoft.Azure.EventHubs.Processor;
-using Microsoft.Azure.WebJobs.EventHubs;
+using Microsoft.Azure.WebJobs.Description;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.Azure.WebJobs.Host.Config;
 using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.EventHubs
```

Comment:
Should this be `...Extensions.EventHubs`?

Reply:
The namespace for this assembly is currently Microsoft.Azure.WebJobs.EventHub which is in agreement with its assembly name. That's a consistency we want to maintain. Ideally the assembly would be Microsoft.Azure.WebJobs.Extensions.EventHubs then the namespace would change to match. If we want to do that assembly rename, we'd do all of this together as a separate change. @paulbatum and I were discussing that yesterday.